### ISSUE TITLE
Hammer of Purity not affected by spell amplification

### ISSUE THREAD

**Issue by FabiJan261** [ValveSoftware/Dota2-Gameplay#2101]
### Ability name

Hammer of Purity

### Description

This ability is not affected by spell amplification from Kaya/Kaya and Sange items. The tooltip shows the newly evaluated Bonus Base Damage value correctly, but no extra damage is actually applied.

### Example Match ID (and possibly Timestamp)

Tested in Demo Mode

**Comment by commentator24** [comment]
what are you talking about..? i just tested in demo and it did apply.
285 lv 30 + no talents
336-341 lv 30 + 50 base 
489-503 lv 30 + 50 base + 75% talent
526-535 with kaya 
577-585 with yasha and kaya

**Comment by FabiJan261** [comment]
Yes, those are the values that are shown by the tooltip, but if you use the ability on an enemy (dummy, creep or hero) the bonus damage by kaya is not actually applied.

**Comment by shaileshgarg** [comment]
I guess it was fixed, currently in Demo mode, Spell Amp on the spell is working fine.
